src/main/java/com/thealgorithms/datastructures/graphs/BipartiteGraphDFS.java

```diff
@@ -50,6 +66,12 @@ public static boolean isBipartite(int v, ArrayList<ArrayList<Integer>> adj) {
         return true;
     }
 
+    /**
```

Comment:
This method can be removed at all. Because better to have tests, which you already added.

Reply:
Removed the `main` method.